concordium-base.cabal

```diff
@@ -473,7 +537,15 @@ benchmark verify-credential-perf
   main-is: haskell-bench/VerifyCredentialPerf.hs
   other-modules:
       Paths_concordium_base
-  default-extensions: FlexibleContexts FlexibleInstances FunctionalDependencies GeneralizedNewtypeDeriving LambdaCase MultiParamTypeClasses RecordWildCards TupleSections
+  default-extensions:
+      FlexibleContexts
+      FlexibleInstances
+      FunctionalDependencies
+      GeneralizedNewtypeDeriving
+      LambdaCase
+      MultiParamTypeClasses
+      RecordWildCards
+      TupleSections
```

Comment:
This reformatting is because you upgraded stack and hpack and this will cause problems for people who have not since stack refuses to regenerate the cabal file, but it is not very clear about it.

Tool upgrades should be planned and, importantly, announced to everybody so that we don't have these failures and noise.

We should also update the static libraries build, and any other place where we pin the stack version, and test that the build still works correctly with the updated version.  This is not a guarantee, it has often failed in the past.